website/docs/admin_hosts_houdini.md

```diff
@@ -3,9 +3,30 @@ id: admin_hosts_houdini
 title: Houdini
 sidebar_label: Houdini
 ---
+## General Settings
+### JOB Path
+
+The Houdini `$JOB` path can be customized through project settings with a (dynamic) path that will be updated on context changes, e.g. when switching to another asset or task.
+
+> If the folder does not exist on the context change it will be created by this feature so that `$JOB` will always try to point to an existing folder.
```

Comment:
Would this be better as as note in the docs?
```suggestion
:::note
If the folder does not exist on the context change it will be created by this feature so that `$JOB` will always try to point to an existing folder.
:::
```

Reply:
That's even better! 
this is not markdown right ? it only works on the website ? 

--- 

I just found that this is supported in markdown  and it's very cool! 
[[Markdown] An option to highlight a "Note" and "Warning" using blockquote (Beta)](https://github.com/orgs/community/discussions/16925)